### PR TITLE
fix Storyshots README, add missing export for image snapshots

### PR DESCRIPTION
**Pull request by thomasbertet**
Issue:

## What I did
- Add missing export to README (& minor fix to other parts of README)

## How to test
- Open the README and read it :smile: 
Is this testable with jest or storyshots?
No
Does this need a new example in the kitchen sink apps?
No
Does this need an update to the documentation?
Yep
If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Review by ndelangen — APPROVED** [review]

**Review by ndelangen — APPROVED** [review]

**Review by Hypnosphi — APPROVED** [review]

**Comment by thomasbertet** [comment]
@ndelangen should I merge or wait for another review ?

**Pull request merged**
